src/api/spec/controllers/webui/obs_factory/staging_projects_controller_spec.rb

```diff
@@ -0,0 +1,133 @@
+require 'rails_helper'
+require 'webmock/rspec'
+# WARNING: If you change owner tests make sure you uncomment this line
+# and start a test backend. Some of the Owner methods
+# require real backend answers for projects/packages.
+# CONFIG['global_write_through'] = true
+
+RSpec.describe Webui::ObsFactory::StagingProjectsController, type: :controller, vcr: true do
+  let(:factory) { create(:project, name: 'openSUSE:Factory') }
+  let!(:factory_staging_a) { create(:project, name: 'openSUSE:Factory:Staging:A', description: '') }
+
+  describe 'GET #index' do
+    let!(:factory_staging) { create(:project, name: 'openSUSE:Factory:Staging') }
+
+    context 'without dashboard package' do
+      before do
+        get :index, params: { project: factory }
+      end
+
+      it { expect(response).to have_http_status(:success) }
+      it { expect(response).to render_template(:index) }
+      it { expect(assigns(:backlog_requests_ignored)).to be_empty }
+      it { expect(assigns(:backlog_requests)).to be_empty }
+      it { expect(assigns(:requests_state_new)).to be_empty }
+      it { expect(assigns(:project)).to eq(factory) }
+    end
+
+    context 'with dashboard package' do
+      let!(:dashboard) { create(:package, name: 'dashboard', project: factory_staging) }
+
+      context 'without ignored_requests file' do
+        before do
+          get :index, params: { project: factory }
+        end
+
+        it { expect(response).to have_http_status(:success) }
+        it { expect(response).to render_template(:index) }
+        it { expect(assigns(:ignored_requests)).to be_nil }
```

Comment:
How is this different to the test above?

Reply:
true, will remove it